OCHuds/Classes/AnimatedBaseHUD.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  AnimatedBaseHUD.swift
+//  Pods
+//
+//  Created by Giovane Berny Possebon on 16/08/16.
+//
+//
+
+import UIKit
+
+public class AnimatedBaseHUD {
```

Comment:
@giovanepossebon do it anyway, and after we think in a better solution for this case.